core/config/value.go

```diff
@@ -421,6 +423,17 @@ func (cv Value) getValueStruct(key string, target interface{}) (interface{}, err
 				continue
 			}
 
+			if fieldType == reflect.TypeOf(_zeroTimeDuration) {
```

Comment:
Yea. Reflect are the expensive calls. It's a micro-optimization, but this is some perf-critical code, so we should aim to reduce expensive calls as much as possible.